dependency_list_export.go

```diff
@@ -0,0 +1,107 @@
+package gitlab
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+)
+
+type DependencyListExportService struct {
+	client *Client
+}
+
+// createDependencyListExportOptions represents the available CreateDependencyListExport()
+// options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/dependency_list_export.html#create-a-pipeline-level-dependency-list-export
+type createDependencyListExportOptions struct {
+	ExportType string `url:"export_type" json:"export_type"`
+}
+
+// DependencyListExport represents a request for a GitLab project's dependency list.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/dependency_list_export.html#create-a-pipeline-level-dependency-list-export
+type DependencyListExport struct {
+	ID          int    `json:"id"`
+	HasFinished bool   `json:"has_finished"`
+	Self        string `json:"self"`
+	Download    string `json:"download"`
+}
+
+// CreateDependencyListExport creates a new CycloneDX JSON export for all the project dependencies
+// detected in a pipeline.
+//
+// If an authenticated user does not have permission to read_dependency, this request returns a 403
+// Forbidden status code.
+//
+// SBOM exports can be only accessed by the export’s author.
+//
+// GitLab docs:
+// https://docs.gitlab.com/ee/api/dependency_list_export.html#create-a-pipeline-level-dependency-list-export
+func (s *DependencyListExportService) CreateDependencyListExport(id int, options ...RequestOptionFunc) (*DependencyListExport, *Response, error) {
```

Comment:
WDYT about being a little bit more explicit with the argument name?

```suggestion
func (s *DependencyListExportService) CreateDependencyListExport(pipelineID int, options ...RequestOptionFunc) (*DependencyListExport, *Response, error) {
```